### PR TITLE
Update dotcms info

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/site/headless-cms/dotcms.md [hunk 1]
```diff
@@ -9,23 +9,28 @@ supportedgenerators:
...
-6. [Best-of-breed technology](https://dotcms.com/marketplace/) - quick and seamless integration with 3rd party systems
-7. [Multi-tenant](https://dotcms.com/landing-pages/case-studies/website-migration-why-junior-achievement-switched-to-dotcms) - manage thousands of sites and channels
-8. [Workflows](https://dotcms.com/product/features/content-workflow) - custom multistep workflows to ensure governance
-9. [SPA Visual Editor](https://dotcms.com/product/technology/api-first-cms#keyfeatures) - drag and drop editor tofor managing layout and design of your single page apps
-10. [GraphQL](https://dotcms.com/product/technology/api-first-cms#keyfeatures) - auto-generated GraphQL schemas based on the content repository
-​
-​
-[More Features](https://dotcms.com/product/features/feature-list) | [Online Demo](https://dotcms.com/demo/)
\ No newline at end of file
+##Get Started
```
Comment (philhawksworth) [inline src/site/headless-cms/dotcms.md:19]: A little markdown fix

```suggestion

## Get Started

```

File: src/site/headless-cms/dotcms.md [hunk 2]
```diff
@@ -9,23 +9,28 @@ supportedgenerators:
...
+- **Docker** - Running dotCMS in Docker is the simplest way to get started with dotCMS on any platform. [Learn More](https://dotcms.com/download/)
+- **Sandbox** - The quickest and easiest way to explore dotCMS is our online demo (admin|admin@dotcms.com). [Learn More](https://demo.dotcms.com/dotAdmin/#/public/login)
+- **Demo** - Schedule time with a dotCMS expert for a personalized demo, tailored to meet your requirements. [Learn More](https://dotcms.com/demo/demo-request)
+- **Online Training** - Free online video training from dotCMS experts who will walk you through building sites in dotCMS. [Learn More](https://dotcms.com/courses/)
+
+##Key Features
```
Comment (philhawksworth) [inline src/site/headless-cms/dotcms.md:25]: Another little markdown fix

```suggestion

## Key Features

```

**Review by philhawksworth — CHANGES_REQUESTED** [review]
Thanks for the update.
Just 2 little markdown fixes needed before we can merge

File: src/site/headless-cms/dotcms.md (hunk 1 above)
Comment (fmontes) [reply src/site/headless-cms/dotcms.md]: Thank you @philhawksworth fixed :)

File: src/site/headless-cms/dotcms.md (hunk 2 above)
Comment (fmontes) [reply src/site/headless-cms/dotcms.md]: Thank you, fixed.

**Pull request merged**